caddy/caddy_test.go

```diff
@@ -573,12 +655,10 @@ func TestAutoWorkerConfig(t *testing.T) {
 			strings.NewReader(expectedMetrics),
 			"frankenphp_total_threads",
 			"frankenphp_busy_threads",
-			"frankenphp_testdata_index_php_busy_workers",
-			"frankenphp_testdata_index_php_total_workers",
-			"frankenphp_testdata_index_php_worker_request_count",
-			"frankenphp_testdata_index_php_worker_crashes",
-			"frankenphp_testdata_index_php_worker_restarts",
-			"frankenphp_testdata_index_php_ready_workers",
+			"frankenphp_busy_workers",
+			"frankenphp_total_workers",
+			"frankenphp_worker_request_count",
+			"frankenphp_ready_workers",
 		))
```

Comment:
It might be a good idea to test what happens if multiple workers have identical names.